x/wasm/keeper/msg_server_integration_test.go

```diff
@@ -265,7 +273,14 @@ func TestRemoveCodeUploadParamsAddresses(t *testing.T) {
 			},
 			uploadConfig:    types.AccessTypeAnyOfAddresses.With(myAddress, otherAddr),
 			expUploadConfig: types.AccessTypeAnyOfAddresses.With(myAddress),
-			expErr:          false,
+		},
+		"remove unknown address does not fail": {
```

Comment:
I add covered it in: "authority cannot remove not existing addresses when permission is any of addresses". We can remove one of them.